src/core/trulens/experimental/otel_tracing/core/instrument.py

```diff
@@ -51,6 +58,10 @@ def wrapper(*args, **kwargs):
                 span.set_attribute(
                     "parent_span_id", parent_span.get_span_context().span_id
                 )
+                span.set_attribute(
```

Comment:
we want app/run id in the baggage as well I guess, but given that it's not totally clear yet no point in doing it now I suppose.

Reply:
I'll put the app id in baggage for now I guess since that seems pretty good to me. For run_id it's a little more ambiguous what the shape of the API will look like so I'll leave that out for now.